scripts/risk_pipeline/data_prep/treat/treatment/twap.py

```diff
@@ -0,0 +1,18 @@
+
+def twap(df, time_col, price_col, tf):
+    # TWAP and periodicity
+    df.index = df[time_col]
+    df.drop(time_col, axis=1, inplace=True)
+    df = df[[price_col]]
+    df = df.rolling(f'{tf}s', min_periods=1).mean()
```

Comment:
what is tf? not a good variable naming choice. I have no idea what this function does